wrappers/javascript/aries-askar-react-native/src/ReactNativeAriesAskar.ts

```diff
@@ -411,29 +427,31 @@ export class ReactNativeAriesAskar implements AriesAskar {
   public async scanStart(options: ScanStartOptions): Promise<ScanHandle> {
     const { category, storeHandle, limit, offset, profile, tagFilter } = serializeArguments(options)
     const handle = await this.promisifyWithResponse<number>((cb) =>
```

Comment:
I see promisifyWithResponse has error handling, and the return value also. Is that because the callback can have an error as well as the ffi call?

Reply:
Yes!